components/LogoNav/LogoNav.tsx

```diff
@@ -11,15 +12,9 @@ import logo from '/public/assets/logo-colored-nav.svg';
  */
 export const LogoNav = (): JSX.Element => {
   return (
-    <Image
-      src={logo}
-      alt="Gridiron Survivor logo"
-      width={1}
-      height={1}
-      priority
-      className="w-20"
-      data-testid="logo-nav"
-    />
+    <Link href="/league/all">
+      <Image src={logo} alt="Gridiron Survivor logo" data-testid="logo-nav" />
```

Comment:
This is a great question. I looked briefly since I'm limited on time this morning and have not found anything. What I did do was added height and width back accordingly to Figma's dimensions:

<img width="1723" alt="image" src="https://github.com/user-attachments/assets/6f60b2a2-1381-4ed5-94dd-81d81945ef2b">

I can definitely look into this more later :)

Reply:
Cool beans. I believe _all_ images need a `width` and `height` declared and that is no different for Next.JS' [Image component](https://nextjs.org/docs/app/api-reference/components/image#props)